README.md

```diff
@@ -125,6 +134,63 @@ environment.
 
 ![](./readme/environment-connection.png)
 
+## <a name="productionPackage"></a>Package the services for production use
+
+The steps described above define how to setup and run the services on a local
+development machine. In order to run the services on a server the following
+procedure should be used.
+
+### Generate a production build and docker image of the backend service
+
+The solution comes with a ready-to-use `Dockerfile` at its root. It requires two
+`ARG`s getting passed in:
+
+- `PACKAGE_ROOT` - needs to be set to the relative path of the service that
+  should be build. This is usually `services/{serviceName}/service` (e.g.
+  `services/media/service`)
+- `PACKAGE_BUILD_COMMAND` - needs to be set to the npm command that should be
+  used to build the service. THis command needs to be defined on the root
+  `package.json`. It is usually `build:{serviceName}-service:prod` (e.g.
+  `build:media-service:prod`)
+
+The process will then use the service's `build` script to create a production
+build of the service and the `start` script to spin up the server, once the
+Docker container starts up.
+
+When running the Docker image, please make sure that you provide the expected
+environment variables to the container, as on a production build it will not use
+the `.env` files.
+
+### Generate the workflows package
+
+To generate a production build of a workflow package run the following command:
+
+```sh
+yarn build:{serviceName}-workflows:prod
+```
+
+This script will create a `.tgz` file on the repository root containing the
+packaged workflows.
+
+To deploy the workflows into an environment run a command like:
+
+```sh
+npx piral-cli@latest publish-pilet --no-fresh --url "https://frontends.service.eu.axinom.net/v1/pilets/{tenantId}/{environmentId}" --api-key "{serviceAccountCredentials}" {packageFile}
+```
+
+Please note, that you need to provide the 'Base64 Encoded Credentials' of a
+service account that has the `Pilets: Publish` permissions as `api-key`. This
+value will be shown to you alongside with the other secrets when creating a
+service account or when generating new secrets for an existing account.
+
+Also make sure that you define all values the pilet expects to receive inside
```

Comment:
maybe we can hint here that these values are generally found in the .env.template file and should be used as KEY/VALUE pairs below?